lib/react_on_rails/configuration.rb

```diff
@@ -11,17 +11,21 @@ def self.configuration
       logging_on_server: true,
       generator_function: false,
       trace: Rails.env.development?,
+      reload_server_js_every_request: Rails.env.development?,
```

Comment:
It should be named:

```
development_mode
```

I think @robwise has a good point. We're not accurate and this is asking the user of the gem to think too much.